### PR TITLE
fix: clicking on points in dot plot fails to select them (PT-187153567)

### PR DESCRIPTION
**Pull request by emcelroy**
PT Story: https://www.pivotaltracker.com/story/show/187153567

### CODE REVIEW

File: v3/src/components/graph/components/graph-component.tsx [hunk 1]
```diff
@@ -65,23 +65,24 @@ export const GraphComponent = observer(function GraphComponent({ tile }: ITileBa
...
-                  pixiPointsRef={pixiPointsRef}
-                />
-              </AxisProviderContext.Provider>
-              <AttributeDragOverlay activeDragId={overlayDragId} />
-            </GraphContentModelContext.Provider>
-          </AxisLayoutContext.Provider>
-        </GraphLayoutContext.Provider>
-      </InstanceIdContext.Provider>
-    </DataSetContext.Provider>
+    pixiPointsRef.current &&
```
Comment (kswenson) [inline v3/src/components/graph/components/graph-component.tsx:68]: Should be able to remove this now, right?

File: v3/src/components/graph/utilities/graph-utils.ts [hunk 2]
```diff
@@ -382,7 +382,7 @@ export interface ISetPointSelection {
 export interface ISetPointCoordinates {
   anchor?: Point
   dataset?: IDataSet
-  pixiPointsRef: IPixiPointsRef
+  pixiPoints: PixiPoints
```
Comment (kswenson) [inline v3/src/components/graph/utilities/graph-utils.ts:385]:
```suggestion
  pixiPoints?: PixiPoints
```
If you make this optional, then you shouldn't need to change as many callers. The code is already testing for existence in line 427 below.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- just a couple of remaining things to clean up a bit.

File: v3/src/components/graph/components/graph-component.tsx (hunk 1 above)
Comment (emcelroy) [reply v3/src/components/graph/components/graph-component.tsx]: Weird. I swear I removed that previously, but I must not have committed the change somehow. 

I retesting things after removing it again and the bug is still fixed.

File: v3/src/components/graph/utilities/graph-utils.ts (hunk 2 above)
Comment (emcelroy) [reply v3/src/components/graph/utilities/graph-utils.ts]: Yes, thanks 👍

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**